test/e2e/tests/tokens/nft/send-nft.spec.js

```diff
@@ -115,13 +115,6 @@ describe('Send NFT', function () {
           tag: 'button',
         });
 
-        // Ensure that this type of NFT is not editable for now
-        // https://github.com/MetaMask/metamask-extension/issues/24320
-        const editButtonPresent = await driver.isElementPresent(
-          '[data-testid="confirm-page-back-edit-button"]',
-        );
-        assert.equal(editButtonPresent, false);
-
         // Confirm the send
```

Comment:
I verified but this does not seems to be issue any more https://github.com/MetaMask/metamask-extension/issues/24320

Can you also plz once verify @sleepytanya